includes/html/table/sensors-common.php

```diff
@@ -94,7 +94,7 @@
 
     $link = generate_url(array('page' => 'device', 'device' => $sensor['device_id'], 'tab' => $tab, 'metric' => $sensor['sensor_class']));
 
-    $overlib_content = '<div style="width: 580px;"><h2>'.$sensor['hostname'].' - '.$sensor['sensor_descr'].'</h1>';
+    $overlib_content = '<div style="width: 580px;"><span class="list-large">'.$sensor['hostname'].' - '.$sensor['sensor_descr'].'</span>';
```

Comment:
hi murrant, 
Why not, same values in these class
```
.list-large {
  font-size: 16px;
  font-weight: bold;
}
```
and 
```
.overlib-text {
  font-weight: bold;
  font-size: 16px;
}
```
but you are right more logical because overlib 👍 

i took exemple on 
```
function onmouseover(event) {
  return overlib('<div class=\'overlib-contents\'><div><span class=\'list-large\' ...
```
in health/metric=storage/view=detail/